src/main/java/org/jboss/msc/Version.java

```diff
@@ -35,40 +33,44 @@
  * @author <a href="mailto:david.lloyd@redhat.com">David M. Lloyd</a>
  */
 public final class Version {
+
     private Version() {}
 
     private static final String JAR_NAME;
     private static final String VERSION_STRING;
 
     static {
-        final Enumeration<URL> resources;
-        String jarName = "(unknown)";
-        String versionString = "(unknown)";
+        String version = "(Unknown)";
+        String jarName = "(Unknown)";
```

Comment:
The case of these should stay as they were with the statements in the original order.